### PR TITLE
feat(issuance): gate deploy and allowlist mutations behind wallet-operation policy

### PR DESCRIPTION
**Pull request by G1de0n**
Follow-up to #1827 and the last piece of the STRIDE remainder for issuance: deploy, allowlist add and allowlist remove were the only custodial issuance execute routes still outside the wallet-operation policy gate.

**What changes**

- `POST /tokens/:tokenId/deploy`, `POST /tokens/:tokenId/allowlist` and `DELETE /tokens/:tokenId/allowlist/:entryId` now run `policyGate` with dedicated extractors and the new operation types `issuance_deploy_execute`, `issuance_allowlist_add_execute` and `issuance_allowlist_remove_execute`, following the merged freeze/pause pattern: the extractor resolves the same wallet the handler will sign with, the handler holds execution to the judged identity (`assertJudgedCustodyWallet` + approved-operation wallet check), `beginApprovedWalletOperationEffect` marks the effect boundary before the first durable write, and deploy fences approved replays whose issuance transaction never settled.
- A token without an on-chain control list mutates only database rows — no custody wallet signs — so its allowlist changes carry no candidate; removing an already-revoked entry stays a no-op acknowledgement.
- Deploy can carry a second signer: a metadata authority wallet distinct from the deployment wallet signs the metadata-URI follow-up. The gate judges one wallet per operation, so the extractor evaluates that wallet's effective policy without persisting an operation and refuses anything short of a plain allow (a secondary signer cannot carry its own approval flow), and the handler pins the metadata signer to the judged identity.
- Deny tests for all four paths (deploy, metadata-wallet deny, allowlist add, allowlist remove) plus claimed-state replay tests (duplicate add conflicts, revoked-entry removal is a silent no-op), and the value-moving conformance inventory gains the three contracts.

**Verification**: issuance suite 252 passed, conformance 44 passed, policy/middleware suites green, openapi spec tests 56 passed (the public policy schema does not enumerate operation types, so no contract change), `tsc --noEmit` and biome clean.

### CODE REVIEW

File: apps/sdp-api/src/routes/issuance/index.ts [hunk 1]
```diff
@@ -301,11 +305,13 @@ issuance.post(
   "/tokens/:tokenId/allowlist",
   requirePermissions("tokens:write"),
   validateBody(addAllowlistSchema),
+  policyGate({ extract: extractAllowlistAddPolicyCandidate }),
   addAllowlistEntry
 );
 issuance.delete(
   "/tokens/:tokenId/allowlist/:entryId",
   requirePermissions("tokens:write"),
+  policyGate({ extract: extractAllowlistRemovePolicyCandidate }),
```
[the remark this answers is not in the document]
Comment (G1de0n) [reply apps/sdp-api/src/routes/issuance/index.ts:314]: Fixed in 3dae3087: the execution envelope now records the method (POST or DELETE) and the full path including the query string, and the replay engine re-issues DELETE without a body — so an approved removal replays against the same route with its signingCustodyWalletId intact. Covered by approved-operation-request.test.ts.

File: apps/sdp-api/src/routes/issuance/handlers/allowlist.ts [hunk 2]
```diff
@@ -275,6 +283,7 @@ export const addAllowlistEntry = async (c: ValidatedBodyContext<typeof addAllowl
         )
       : null;
 
+    await beginApprovedWalletOperationEffect(c);
```
[the remark this answers is not in the document]
Comment (G1de0n) [reply apps/sdp-api/src/routes/issuance/handlers/allowlist.ts]: Fixed in 3dae3087: database-mode add and remove now commit the effect fence and the first durable allowlist mutation in one runApprovedWalletOperationEffectTransaction, so a crash between them retries instead of requiring manual reconciliation. The on-chain mode keeps the fence directly before the chain effect, same as the freeze/pause pattern.

**Review by GuiBibeau — APPROVED** [review]
